code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -24,12 +24,35 @@
 	var/isbroken = 0
 	light_range = 5
 	light_color = LIGHT_COLOR_RED
-	var/obj/item/wepon = null
+	var/last_check = 0
+
+/obj/structure/cult/pylon/New()
+	..()
+	processing_objects.Add(src)
+
+/obj/structure/cult/pylon/Destroy()
+	..()
```

Comment:
Destroy inheritance goes in reverse - this should come after removing the object from processing.